frontend/src/features/books/EditBookForm.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
```

Comment:
@nshelokar-vf, do we really need to import `React`?
Please check

Reply:
I checked and no need to import it.